bioptim/examples/getting_started/example_multiphase_test.py

```diff
@@ -0,0 +1,190 @@
+"""
+This example is a trivial box that must superimpose one of its corner to a marker at the beginning of the movement and
+a the at different marker at the end of each phase. Moreover a constraint on the rotation is imposed on the cube.
+Finally, an objective for the transition continuity on the control is added. Please note that the "last" control
+of the previous phase is the last shooting node (and not the node arrival).
+It is designed to show how one can define a multiphase optimal control program
```

Comment:
I don't get the name of the file `example_multiphase_test`? As a user, what should I expect that this example to showcase? Test of multiphase? What does that mean?

Reply:
Also, please update the description to explain what this example specifically showcases